### PR TITLE
Specify which user roles can perform which actions

### PR DESCRIPTION
**Pull request by notnamed**
Updating the actions page to reflect the recent role changes that grant additional functionality and/or expose existing functionality in the UI and API to Operator and Developer users.

Change-type: minor
Signed-off-by: Jordan Cooks jordan@balena.io

### CODE REVIEW

File: pages/learn/manage/actions.md [hunk 1]
```diff
@@ -175,3 +179,4 @@ __Warning:__ It is a good idea to [move your devices to another application][mov
 [developer]:/learn/manage/account/#developer
 [support-access]:/learn/manage/support-access
 [inactive-devices]:/learn/manage/billing/#inactive-devices
+[application-members]:learn/manage/account/#application-members
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:182]:
```suggestion
[application-members]:/learn/manage/account/#application-members
```

File: pages/learn/manage/actions.md [hunk 2]
```diff
@@ -60,6 +62,8 @@ __Warning:__ This action is only supported on devices with an Agent version >= 1
 
 ## Device-specific actions
 
+[Application members][application-members] with the Operator role and above can perform any of the actions listed below.
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:65]:
```suggestion
> [Application members][application-members] with the Operator role and above can perform any of the actions listed below.
```

Converting this to a quote might seem better

File: pages/learn/manage/actions.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ Actions allow you to control the status of your applications and devices during
 
 ## General actions
 
+[Application members][application-members] with the Operator role and above can perform any of the actions listed below.
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:15]:
```suggestion
> [Application members][application-members] with the Operator role and above can perform any of the actions listed below.
```

Converting this to a quote

File: pages/learn/manage/actions.md [hunk 4]
```diff
@@ -116,7 +120,7 @@ __Warning:__ Only change the device type if a device was incorrectly provisioned
 
 ## Application-specific actions
 
-These actions can be found on the "Actions" menu for each application and apply to the application and all the devices in the fleet.
+These actions can be found on the "Actions" menu for each application and apply to the application and all the devices in the fleet. [Application members][application-members] with the Developer role and above can perform any of the actions listed below.
```
Comment (Page-) [inline pages/learn/manage/actions.md:123]: Can developers transfer ownership?

Comment (notnamed) [reply pages/learn/manage/actions.md:123]: As far as I can see, yes - @thgreasi can you confirm?

**Comment by vipulgupta2048** [comment]
@notnamed can you add the changes Thodoris suggested

File: pages/learn/manage/actions.md (hunk 4 above)
Comment (vipulgupta2048) [reply pages/learn/manage/actions.md:123]: Made the change, resolving. Admin it is.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**